packages/modelviewer.dev/data/examples.json

```diff
@@ -31,19 +31,19 @@
     "examples": [
       {
         "htmlId": "demo-container-1",
-        "name": "Autoplay animation"
+        "name": "Autoplay"
       },
       {
         "htmlId": "demo-container-2",
-        "name": "Select specific"
+        "name": "Selecting Animations"
       },
       {
         "htmlId": "demo-container-3",
-        "name": "Animation crossfade"
+        "name": "Crossfade"
```

Comment:
I think the bullet could be removed and replaced with a link that just says "Examples", unless you have plans for a bunch of unique wording there.

Reply:
Maybe "Related Examples"

Reply:
I was thinking that some might have a specific example to reference, as opposed to Examples as a whole

Reply:
Yeah, I agree that linking to a specific example would be good, but I'm not sure that specific text is required. "Related" is probably a good enough indicator. Less per-attribute text to manage is a good thing 👍

Reply:
roger roger